css/main.css

```diff
@@ -1165,6 +1165,19 @@ li.tg-rss a{background:#ff6600;}
 	width:38px;
 	line-height:38px;
 }
+
+@media screen and (max-width: 767px) {
```

Comment:
@CuriousLearner  How about "Responsive utilities" of bootstrap? We can edit the sizes using visible or hidden classes.